app/models/clinic.rb

```diff
@@ -15,11 +17,14 @@ class Clinic
   field :state, type: String # ennnnnnummmmerrrrattttttioonnn???????
   field :zip, type: String
 
+<<<<<<< HEAD
```

Comment:
Go ahead and remove lines 20-29 here, since we're moving that to the `Auditable` concern.